docs/overview/renderers/nema_gfx.rst

```diff
@@ -0,0 +1,73 @@
+.. _stm32_nema_gfx:
+
+=============================================
+STM32 NemaGFX GPU (AKA NeoChrom or Chrom-ART)
```

Comment:
1. NeoChrom refers to NemaP GPU (Made by TSi)
2. NeoChromVG refers to NemaPVG GPU (Made by TSi)
3. Chrom-Art is the legacy ST system

Reply:
I noticed and fixed my mistake with Chrom-ART before seeing your comment. Chrom-ART is DMA2D.

In your opinion, what would the best title for this doc be?